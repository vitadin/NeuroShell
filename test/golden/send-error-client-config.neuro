%% Send command error test - Client configuration error
%% Tests that \send command handles client configuration errors properly
%% This verifies that errors are displayed at the end after any available content

\set _style dark1

\echo === Send Client Configuration Error Test ===
\echo Testing \send command with client configuration error

%% Test successful message first
\echo Testing successful message...
\send This message should work fine.

%% Test message that triggers client configuration error
\echo Testing client configuration error...
\send Please trigger client error to test error handling.

%% Test recovery
\echo Testing recovery after client error...
\send This should work normally after the error.

%% Show final state
\echo Checking final session state:
\session-show

%% Verify error handling variables
\echo Error handling verification:
\get[#llm_error_code]
\echo Last error code: ${#llm_error_code}
\get[#llm_error_message] 
\echo Last error message: ${#llm_error_message}