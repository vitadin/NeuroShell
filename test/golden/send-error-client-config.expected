Setting _style = dark1
=== Send Client Configuration Error Test ===
Testing \send command with client configuration error
Testing successful message...
<thinking id="2-1">
Thinking about the user's message: "This message should work fine.". This helps verify the message flow in tests. The user sent 1 messages total, and I need to provide a helpful response.
</thinking>

  This is a mocking reply (received 1 messages, last: This message should work
  fine.)                                                                      

Testing client configuration error...
FATA Script execution failed error="command execution failed: LLM call failed: Mock client configuration error for testing"